.github/workflows/test.yml

```diff
@@ -40,12 +40,12 @@ jobs:
       - name: sampling-test
         shell: bash -l {0}
         run: |
-          svgdigitizer digitize test/data/sampling.svg --sampling_interval 1
+          svgdigitizer digitize test/data/sampling.svg --sampling_interval .00101
```

Comment:
These constants were chosen such that the output CSV has the same number of rows than before.